lib/screens/surveydetails/widget/question_paging_widget.dart

```diff
@@ -28,80 +30,96 @@ class _QuestionPagingWidgetState extends State<QuestionPagingWidget> {
-        )
+        ),
       ],
     );
   }
 
+  Widget _buildPagedQuestions(
+    BuildContext context,
+    PageController pageController,
+  ) {
+    return Expanded(
+      child: PageView(
+        controller: pageController,
+        onPageChanged: _onPageChanged,
+        physics: const NeverScrollableScrollPhysics(),
+        children: widget.surveyDetailUiModel.questions
+            .map((QuestionUiModel question) =>
+                _buildQuestionHeader(context, question))
+            .toList(),
+      ),
+    );
+  }
+
   Widget _buildQuestionToolbar(BuildContext context, QuestionUiModel question) {
-    return Row(
-      children: [
-        _isIntroQuestion(question)
-            ? MaterialButton(
-                onPressed: () => {context.pop()},
-                minWidth: 30,
-                height: 30,
-                materialTapTargetSize: MaterialTapTargetSize.shrinkWrap,
-                padding: EdgeInsets.zero,
-                child: SvgPicture.asset(Assets.images.icBack),
-              )
-            : const SizedBox(),
-        const Spacer(),
-        !_isIntroQuestion(question)
-            ? ElevatedButton(
-                onPressed: () => {context.pop()},
-                style: ElevatedButton.styleFrom(
-                  shape: const CircleBorder(),
-                  padding: const EdgeInsets.all(8.5),
-                  backgroundColor: Colors.white.withOpacity(0.2),
-                ),
-                child: Icon(
-                  Icons.close,
-                  color: Theme.of(context).primaryColor,
-                ),
-              )
-            : const SizedBox(),
-      ],
+    return Padding(
+      padding: const EdgeInsets.only(bottom: Dimensions.paddingMedium),
+      child: SizedBox(
+        height: 44.0,
+        child: Row(
+          children: [
+            _isIntroQuestion(question)
+                ? MaterialButton(
+                    onPressed: () => context.pop(),
+                    minWidth: 30,
+                    height: 30,
+                    materialTapTargetSize: MaterialTapTargetSize.shrinkWrap,
+                    padding: EdgeInsets.zero,
+                    child: SvgPicture.asset(Assets.images.icBack),
+                  )
+                : const SizedBox(),
+            const Spacer(),
+            !_isIntroQuestion(question)
+                ? MaterialButton(
+                    onPressed: () => context.pop(),
+                    minWidth: 28,
+                    height: 28,
+                    materialTapTargetSize: MaterialTapTargetSize.shrinkWrap,
+                    color: Colors.white.withOpacity(0.2),
+                    textColor: Theme.of(context).primaryColor,
+                    padding: EdgeInsets.zero,
+                    shape: const CircleBorder(),
+                    child: const Icon(
+                      Icons.close,
+                      size: 18,
+                    ),
+                  )
+                : const SizedBox(),
```

Comment:
Can we just use `if (...) Widget()`, so we don't need to build SizedBox()
```suggestion
            if (!_isIntroQuestion(question))
                MaterialButton(
                    onPressed: () => context.pop(),
                    minWidth: 28,
                    height: 28,
                    materialTapTargetSize: MaterialTapTargetSize.shrinkWrap,
                    color: Colors.white.withOpacity(0.2),
                    textColor: Theme.of(context).primaryColor,
                    padding: EdgeInsets.zero,
                    shape: const CircleBorder(),
                    child: const Icon(
                      Icons.close,
                      size: 18,
                    ),
                  ),
```